packages/jellyfish-api-core/__tests__/category/account/transferDomain.test.ts

```diff
@@ -527,17 +527,18 @@ describe('TransferDomain', () => {
       await expect(promise).rejects.toThrow('amount 90000.00000000 is less than 999999.00000000')
     })
 
-    it('(evm -> dvm) should fail if insufficient balance', async () => {
+    // TODO(canonbrother): check again here after pull the later version
```

Comment:
still need to check this? @canonbrother

Reply:
making a smaller scope with this specific fix on another PR is preferable.. we need a checkpoint first

Reply:
next check will be on `HEAD-53859f66f9`, the rc pre-release https://github.com/DeFiCh/ain/pull/2485
why we do a checkpoint here.. as this `master-b352814976` is the most stable version that i found while compare others ain against Jellyfish
we wouldn't know will the later regression cause other non-related test failed and doing kick and roll CI running or a real issue on evm which take days to get a release which's killing productivity on Jellyfish side

maybe take this as jellyfish evm compatible PR 🤔

Reply:
got it